contracts/pcv/PCVSwapperUniswap.sol

```diff
@@ -85,11 +85,22 @@ contract PCVSwapperUniswap is IPCVSwapper, OracleRef, Timed {
         _initTimed();
     }
 
-    /// @notice All received ETH is wrapped to WETH
-    receive() external payable {
-      if (msg.sender != address(WETH)) {
-        WETH.deposit{value: msg.value}();
-      }
+    /// @notice Empty callback on ETH reception
+    receive() external payable {}
+
+    // =======================================================================
+    // WETH management
+    // =======================================================================
+
+    /// @notice Wraps all ETH held by the contract to WETH
+    function wrapETH() external onlyPCVController {
+        IWETH(WETH).deposit{value: address(this).balance}();
+    }
+
+    /// @notice Unwraps all WETH held by the contract back to ETH
+    function unwrapETH() external onlyPCVController {
+        uint256 amount = ERC20(WETH).balanceOf(address(this));
+        IWETH(WETH).withdraw(amount);
     }
```

Comment:
Lets make wrapETH external with no access control and keep the old behavior of `withdrawETH` where it only unwraps the amount desired.

Intended design is that all ETH sent in can become WETH and swap intentionally. And WETH can only be unwrapped and leave on withdrawETH calls by a PCVController